src/styles/blocks/button.scss

```diff
@@ -0,0 +1,35 @@
+.button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  width: 100%;
+  height: 56px;
+  background-color: #292929;
```

Comment:
This color is heavily reused in your stylings, it could be a variable (generally all your colors ideally would be variables)